examples/automm/kaggle_feedback_prize/README.md

```diff
@@ -0,0 +1,162 @@
+# Using MultiModalPredictor with Text Normalization
+
+Take [Feedback Prize - Predicting Effective Arguments](https://www.kaggle.com/competitions/feedback-prize-effectiveness) as an example to show how text normalization can be helpful.
+
+## 1. Preprocess data with text normalization
+
+Text normalization is the task of mapping non-canonical language, typical of speech transcription and computer-mediated communication, to a standardized writing. 
+It is an up-stream task necessary to enable the subsequent direct employment of standard natural language processing tools and indispensable for languages such as Swiss German, 
+with strong regional variation and no written standard. 
+Even though the competition dataset is composed of English only, we found that applying text normalization can reduce `log loss`, the metrics that this competition is evaluating on. 
+
+### 1.1 Define error handlers for codecs
+    def replace_encoding_with_utf8(error: UnicodeError) -> Tuple[bytes, int]:
+        return error.object[error.start : error.end].encode("utf-8"), error.end
+
+    def replace_decoding_with_cp1252(error: UnicodeError) -> Tuple[str, int]:
+    return error.object[error.start : error.end].decode("cp1252"), error.end
+
+
+### 1.2 Register error handlers for codecs
+    codecs.register_error("replace_encoding_with_utf8", replace_encoding_with_utf8)
+    codecs.register_error("replace_decoding_with_cp1252", replace_decoding_with_cp1252)
+
+
+### 1.3 Applying a series of decoding and encoding for normalization
+    def resolve_encodings_and_normalize(text: str) -> str:
+        text = (
+            text.encode("raw_unicode_escape")
+            .decode("utf-8", errors="replace_decoding_with_cp1252")
+            .encode("cp1252", errors="replace_encoding_with_utf8")
+            .decode("utf-8", errors="replace_decoding_with_cp1252")
+        )
+        text = unidecode(text)
+        return text
+
+### 1.4 Applying normalization to feature columns
+    def read_and_process_data_with_norm(path: str, file: str, is_train: bool) -> pd.DataFrame:
+        df = pd.read_csv(os.path.join(path, file))
+        df["discourse_text"] = df["discourse_text"].apply(resolve_encodings_and_normalize)
+        return df
+
+For details, please refer to 
+[`kaggle_feedback_prize_preprocess.py`](./kaggle_feedback_prize_preprocess.py).
+
+## 2. MultiModalPredictor for Training
+
+MultiModalPredictor can automatically build deep learning models with multimodal datasets. 
+The tabular data we have for this Kaggle competition is a perfect example that showcases how easily we could build models with just a hundred lines using MultiModalPredictor. 
```

Comment:
May not emphasize “a hundred lines” here. It may make people feel that they still need to write lots of code.

Reply:
Updated. It's at most a couple dozens of lines :)